code/extension/ShareThisSTE.php

```diff
@@ -91,50 +124,67 @@ public function ShowShareIcons()
         }
     }
 
-    public function ShareIcons()
+    /**
+     * Get the sharing icons
+     */
+    public function getShareIcons()
     {
         $bookmarks = $this->makeBookmarks('IncludeThisIcon');
         return $this->makeShareIcons($bookmarks);
     }
 
+    /**
+     * Grabbing front end dependencies for the expanded sharing list with some extra
+     * functionality
+     */
     public function ShareAllExpandedList()
     {
-        Requirements::javascript(FRAMEWORK_DIR . '/thirdparty/jquery/jquery.js');
+        Requirements::javascript('/vendor/silverstripe/admin/thirdparty/jquery/jquery.js');
```

Comment:
This vendor file cannot be downloaded in the browser as it comes back with a forbidden access error. The links to these files should be to the versions that are exposed in the resources folder.

Reply:
Done!